site/frontend/src/pages/compare/tabs.vue

```diff
@@ -95,36 +98,33 @@ const activeTab: Ref<Tab> = ref(props.initialTab);
 
 <template>
   <div class="wrapper">
-    <div
-      class="tab"
-      title="Compilation time benchmarks: measure how long does it take to compile various crates using the compared rustc."
-      :class="{selected: activeTab === Tab.CompileTime}"
+    <TabComponent
+      :tooltip="'Compilation time benchmarks: measure how long does it take to compile various crates using the compared rustc.'"
+      :title="'Compile-time'"
+      :selected="activeTab === Tab.CompileTime"
       @click="changeTab(Tab.CompileTime)"
     >
-      <div class="title">Compile-time</div>
-      <div class="summary table-wrapper">
+      <template v-slot:summary>
         <SummaryTable :summary="compileTimeSummary" />
-      </div>
-    </div>
-    <div
-      class="tab"
-      title="Runtime benchmarks: measure how long does it take to execute (i.e. how fast are) programs compiled by the compared rustc."
-      :class="{selected: activeTab === Tab.Runtime}"
+      </template>
+    </TabComponent>
+    <TabComponent
+      :tooltip="'Runtime benchmarks: measure how long does it take to execute (i.e. how fast are) programs compiled by the compared rustc.'"
+      :title="'Runtime'"
```

Comment:
```suggestion
      title="Runtime"
```
If you don't use `:` before the name of the attribute, it will be evaluated as a string, so you won't need the extra apostrophes. (same for the other `:title` attributes).